### PR TITLE
Different loss weight for edit tags

### PR DESCRIPTION
**Pull request by yechenh**

### CODE REVIEW

File: lasertagger/run_lasertagger.py [hunk 1]
```diff
@@ -127,6 +127,11 @@
 flags.DEFINE_string("embedding_type", None, "Type of embedding. ['Normal', 'POS', " 
                     "'POS_concise', 'Sentence']")
 
+
+flags.DEFINE_float("add_tag_loss_weight", 1, "The weight of loss of add tags.")
```
Comment (XiangZeng2020) [inline lasertagger/run_lasertagger.py:131]: s/The weight of loss of add tags/The weight of loss for adding tags

ditto

File: lasertagger/run_lasertagger_utils.py [hunk 2]
```diff
@@ -323,3 +350,33 @@ def _get_decoder_params(config, use_tpu,
...
+    """
+    if weight == 1:
+        return loss
+    else:
+        filter_label_mask = tf.constant(0.0, dtype="float32", shape=labels.get_shape())
+        for filter_label in filter_labels:
+            filter_label_mask = tf.math.add(
+                tf.cast(tf.math.equal(tf.constant(filter_label, dtype="int32"), labels), tf.float32), 
+                filter_label_mask)
+        loss_weight = tf.math.scalar_mul(tf.constant(weight - 1, dtype="float32"), filter_label_mask)
```
Comment (XiangZeng2020) [inline lasertagger/run_lasertagger_utils.py:375]: why weight - 1 ?

Comment (yechenh) [reply lasertagger/run_lasertagger_utils.py:375]: I add that 1 back on line 378-379. The idea is that for all the elements in the loss tensor whose corresponding element is 0 in the filter_label_mask, I keep that element the same. For all elements in the loss tensor whose corresponding element in the filter_label_mask is 1, I multiply that element by the weight. So basically I need a tensor where all elements are 1 except a few that need to be weighted by the weight. And line 374-375 along with line 378-379 is constructing such tensor.

File: lasertagger/run_lasertagger.py (hunk 1 above)
Comment (yechenh) [reply lasertagger/run_lasertagger.py]: Done

File: lasertagger/run_lasertagger_utils.py (hunk 2 above)
Comment (XiangZeng2020) [reply lasertagger/run_lasertagger_utils.py:375]: Got it, thanks for the explanation.

**Pull request merged**